doc/architecture/sensor_view_configuration.adoc

```diff
@@ -0,0 +1,15 @@
+= Sensor view configuration
+
+The sensor view configuration configures the sensor view.
+There are two sources for sensor-view-configuration data.
+
+Sensor-view-configuration data can be provided by the environment simulation.
```

Comment:
```suggestion
Sensor-view-configuration data can be provided by the simulation environment.
```
Or to the "environment simulator"

Reply:
CCB Meeting 04.08.2021:
- rejected as per discussion.